batchlib/workflows/cell_analysis.py

```diff
@@ -213,15 +216,34 @@ def core_workflow_tasks(config, name, feature_identifier):
               'actual_channels_to_use': (*serum_ana_in_keys, marker_ana_in_key),  # is actually used
               'feature_identifier': feature_identifier,
               'cell_seg_key': config.seg_key}}),
+         (ErodeSegmentation, {
```

Comment:
It would be better if we don't hard-code, but it's ok for today and I will clean up tomorrow.